libs/python/detect.go

```diff
@@ -32,3 +46,29 @@ func DetectExecutable(ctx context.Context) (string, error) {
 	}
 	return interpreter.Path, nil
 }
+
+// DetectVEnvExecutable returns the path to the python3 executable inside venvPath,
+// that is not necessarily activated.
+//
+// If virtualenv is not created, or executable doesn't exist, the error is returned.
+func DetectVEnvExecutable(venvPath string) (string, error) {
+	interpreterPath := filepath.Join(venvPath, "bin", "python3")
+	if runtime.GOOS == "windows" {
+		interpreterPath = filepath.Join(venvPath, "Scripts", "python3.exe")
+	}
+
+	if _, err := os.Stat(interpreterPath); err != nil {
+		if os.IsNotExist(err) {
+			return "", fmt.Errorf("can't find %q, check if virtualenv is created", interpreterPath)
+		} else {
+			return "", fmt.Errorf("can't find %q: %w", interpreterPath, err)
+		}
+	}
+
+	_, err := os.Stat(filepath.Join(venvPath, "pyvenv.cfg"))
```

Comment:
Please include a link with details describing how this is created and what it contains.

Reply:
This is a good article: https://snarky.ca/how-virtual-environments-work/

The file contains information like the Python version and how to inherit the system environment. Initially, I wrote a parser for it to extract the version, but we don't really need it, so I erased it.